lib/sonata/tip/executeUserTip.ts

```diff
@@ -15,22 +19,26 @@ async function executeUserTip(
 
   const { allowableAmount: amount, tipperFid: sender, tip, channelTip } = tipInfo;
   let receiverAmount = amount;
-  const stackCalls = [];
+  const stacks = [];
   const allUpdates = [];
 
   if (channelTip) {
     const { channelAddress, channelAmount, channelId } = channelTip;
-    stackCalls.push(stack.track(`channel_tip_from_${sender}`, { account: channelAddress, points: channelAmount }));
+    stacks.push(stack.track(`channel_tip_${channelId}`, { account: channelAddress, points: channelAmount }));
```

Comment:
why are we changing the event name?
- this will make it harder to track legacy tips right?

Reply:
We currently track tips via **supabase** entries, will not be an issue for legacy tips.
If we need total tip points by channel id from the `stack`, it is impossible with previous events. Now it can be done with `channel_tip_${channelId}`